viam-cartographer_test.go

```diff
@@ -356,6 +409,30 @@ func TestSLAMProcess(t *testing.T) {
 		grpcServer.Stop()
 	})
 
+	t.Run("Successful reconfigure of SLAM process results in new timestamp", func(t *testing.T) {
+		grpcServer, port := setupTestGRPCServer(t)
+		attrCfg := &vcConfig.Config{
+			Sensors:       []string{},
+			ConfigParams:  map[string]string{"mode": "2d", "test_param": "viam"},
+			DataDirectory: dataDir,
+			Port:          "localhost:" + strconv.Itoa(port),
+			UseLiveData:   &_false,
+		}
+
+		svc, err := testhelper.CreateSLAMService(t, attrCfg, logger, false, testExecutableName)
+		test.That(t, err, test.ShouldBeNil)
+
+		timestamp1, _ = svc.GetLatestMapInfo(context.Background())
+		svc.Reconfigure(context.Background(), resource.Dependencies{}, resource.Config{})
+		timestamp2, _ = svc.GetLatestMapInfo(context.Background())
+		test.That(t, timestamp1, test.ShouldNotEqual, &_zeroTime)
```

Comment:
You can simply these by using a test.ShouldBeBetween

Reply:
I tried that, but because they are of type time.Time, the test package doesn't know how t compare them